logstreams/src/test/java/io/zeebe/logstreams/util/LogStreamRule.java

```diff
@@ -90,7 +90,7 @@ protected void after() {
     actorSchedulerRule.after();
   }
 
-  private void startLogStream() {
+  public void createLogStream() {
```

Comment:
does it need to be public available?

Reply:
Yes. For access in  `LogStreamTest#shouldIncreasePositionOnRestart`